priya/javascript/Assignment11/JsAssign11Q1.html

```diff
@@ -0,0 +1,66 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script>
+      function show_roll(sorted_mark) {
```

Comment:
Change function to empty parameter
```suggestion
      function show_roll() {
```